server/database/scraping_pipeline.py

```diff
@@ -0,0 +1,61 @@
+"""
+Pipeline checks which data to scrape, scrapes data and saves relevant data.
+"""
+from play_store_scraper_facillitator import PlayStoreScraperFacillitator
+from play_store_textpreprocess import preprocess_playstore_data
+from app_store_scraper_facillitator import AppScraperFacillitator
+from app_store_textpreprocess import preprocess_appstore_data
+
+class DataPipeline:
+
+    def __init__(self):
+        """
+        Initialise DataPipeline object
+        On initialise, extract relevant data from database
+
+        Attr:
+            bank_names (list): List of bank names
+            playstore_apps (dict): Dictionary of bank names to playstore apps
+            playstore_app_ids (dict): Dictionary of bank names to playstore app ids
+            appstore_apps (dict): Dictionary of bank names to appstore apps
+            last_scraped (datetime): datetime of last called scrape
+        """
+        bank_names = [] # Pull from database
+        playstore_apps = {} # Pull from database
+        playstore_app_ids = {} #Pull from database
+        appstore_apps = {} # Pull from database
+        tokens = {} # Pull from database
+        last_scraped = None # Pull from datapase
+    
+    def update_reviews(self):
+        """
+        Get preprocessed reviews from playstore and appstore update to database
+        """
+        playstore_scraper = PlayStoreScraperFacillitator(self.bank_names, self.playstore_apps, self.playstore_app_ids)
```

Comment:
Why is there an additional `playstore_app_ids` parameter added to this facilitator?